data/docs/gcp-monitoring/gke-tracing.mdx

```diff
@@ -0,0 +1,1190 @@
+* [Install Helm](https://helm.sh/docs/intro/install/)
+* [Self-Hosted SigNoz in GKE Cluster](https://signoz.io/docs/install/kubernetes/gcp/) ( If you are hosting SigNoz in Google Kubernetes Engine (GKE), you should refer to the specific guide that outlines the minimum requirements and provides details for the override-values.yaml ) OR if you’re hosting SigNoz on VM, follow [Self-Hosted SigNoz on VM ](https://signoz.io/docs/install/docker/).
+
+## Quick Start
+
+Check that the nodes in your GKE clusters are in `Ready` state:
+
+<figure data-zoomable align="left">
+    <img
+        src="/img/docs/gcp-monitoring/gke-tracing/cluster-status.webp"
+        alt="GKE Cluster Status"
+    />
+<figcaption>
+<i>
+GKE Cluster Status
+</i>
+</figcaption>
+</figure>
+
+**Step 1:** Add the SigNoz helm repo using this command
+
+```sh
+helm repo add SigNoz [https://charts.signoz.io](https://charts.signoz.io)
+```
+
+<Tabs>
+<TabItem value="self-hosted-on-gke-cluster" label="For Self-hosted SigNoz in GKE Cluster" default>
+
+**Step 2:** Install the OTel Collector agent in GKE where SigNoz is running in a Self-hosted VM.
+
+```sh
+helm install -n signoz  --create-namespace kubelet-otel signoz/k8s-infra \\
+otelCollectorEndpoint="<self-host-signoz-otel-collector-service-fqdn>:4317 " --set otelInsecure=true
+```
+
+</TabItem>
+<TabItem value="self-host-signoz-in-vm" label="For Self-hosted SigNoz in a VM.">
+
+**Step 2:** Install the OTel Collector agent in GKE where SigNoz is running in a Self-hosted VM.
+
+```sh
+helm install -n signoz  --create-namespace kubelet-otel signoz/k8s-infra \\
+otelCollectorEndpoint="<self-host-signoz-vm-ip>:4317" --set otelInsecure=true
+```
+<Admonition type="note">
+Depending on how you have set up your infrastructure, the `otelCollectorEndpoint` can be different from the VM IP.
+</Admonition>
+
+</TabItem>
+</Tabs>
+
+After applying the above commands, check whether the pods are running successfully or not
+
+<figure data-zoomable align="left">
+    <img
+        src="/img/docs/gcp-monitoring/gke-tracing/signoz-status.webp"
+        alt="Resources Status"
+    />
+<figcaption>
+<i>
+Resources Status
+</i>
+</figcaption>
+</figure>
+
+**Step 3:** Make the appropriate changes in the OTel collector config map.
+
+Use the following command to get config maps in the `signoz` namespace
+
+```sh
+kubectl get configmap -n signoz
+```
+
+<figure data-zoomable align="left">
+    <img
+        src="/img/docs/gcp-monitoring/gke-tracing/config-map.webp"
+        alt="OTel configmaps"
+    />
+<figcaption>
+<i>
+OTel configmaps
+</i>
+</figcaption>
+</figure>
+
+Use the following command to edit the config map:
+
+```sh
+kubectl edit configmap -n signoz <config-map name>
+```
+
+Add the following configurations in the config map.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+  name: kubelet-otel-k8s-infra-otel-agent
+data:
+  otel-agent-config.yaml: |2-
+
+    exporters:
+      otlp:
+        endpoint: "<App-endpoint>"
```

Comment:
We;ll need same placeholder here and https://github.com/SigNoz/signoz-web/pull/599/files#diff-0d976b600f5acc59d180ed23cab10aabf1efc7fbe8e9294890c9782b2a8ce153R1082-R1083

Reply:
fixed